_includes/release.html

```diff
@@ -131,6 +131,9 @@ <h3 class="section">Tested Versions</h3>
                   {% else %}
                     <span class="test-with-subcategory"> JDBC Driver: </span> {{ driverVersions  | join: ", "  }} <br />
                   {% endif %}
+                  {% if integration.name == 'Vitess' %}
```

Comment:
How about checking for the "note" value being non-empty instead? That way, it should work for other connectors, too, and we'd avoid the empty "*:" for Vitess 1.4.

Reply:
Great suggestion! Addressed it in the code. Now the rendered page looks like the following:
![image](https://user-images.githubusercontent.com/1978545/107976732-b242e880-6fc2-11eb-8165-8126e644156d.png)